### DISCUSSION TITLE
Any way to use 1.2 instead of 1.3?

### DISCUSSION THREAD

**Discussion by 0x14307** [microsoft/WindowsAppSDK#3898] [Q&A]
Hi there,

I'm using the Windows App SDK along with WinUI 3 to build an app and am running with into an issue trying to get the runtime installed. It installs fine under the 1.2 runtime so is there any way I can use that instead of the 1.3 runtime?

**Comment by DarranRowe** [comment]
Yes, the guaranteed way is to rebuild your application while referencing the 1.2 NuGet package. If your application is packaged then this reference is made at build time and you would need to remake the package. If your application is unpackaged, then it would be the bootstrapper that determines the reference, and this is hard coded if you use WindowsPackageType = None.

There is a possible way to do this with an unpackaged application, but this is likely completely unsupported. Any attempt to use features in 1.3 that are not available in 1.2 would require version checks. The calls would be unsuccessful if you just outright called it.
